internal/pkg/template/diff/diff.go

```diff
@@ -90,7 +90,77 @@ func isYAMLLeaf(node *yaml.Node) bool {
 }
 
 func parseSequence(from, to *yaml.Node) (map[string]*Node, error) {
-	return nil, nil
+	fromSeq, toSeq := make([]yaml.Node, len(from.Content)), make([]yaml.Node, len(to.Content))
+	if err := to.Decode(&toSeq); err != nil {
```

Comment:
do you have to pass in a slice with the capacity already allocated for this to work? or does `Decode` already call `make()`?

Reply:
I *think* `Decode` calls `make()` if it's a `nil` slice, and does not do so otherwise. The motivation was to allocate just enough memory to prevent resizing the slice.

Reply:
i wonder if it'd be nice to just rely on `Decode` that bc i don't think perf/space is that big of an issue and i'd like to simplify where we can...at least until we have evidence that resizing is causing noticeable perf issues

Reply:
done!